app-text/cutemarked/cutemarked-0.11.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A Qt-based Markdown editor"
+DESCRIPTION="CuteMarkEd is a Qt-based Markdown editor with live HTML preview, \
++math expressions, code and markdown syntax highlighting."
```

Comment:
Oops,  "+math" -> "math"

BTW, the app is working fine on x86_64 but on x86_gcc2 I don't have any window displayed.
What arch are you using? If you are on x86_gcc2, does the app work for you on that arch?

Maybe we should mark it as untested on x86 secondary arch:
```diff
-SECONDARY_ARCHITECTURES="x86"
+SECONDARY_ARCHITECTURES="?x86"
```
If someone finds out that it is working fine on that arch then we can always drop the "?" afterwards.

Reply:
I'm also using x86_64. I think app doesn't work on gcc2 because qt5 is broken on that arch.

Reply:
OK.
BTW, I have 2 VMs running in parallel with Xen. And they share the same haikuports tree, so I can easily test any recipe on all architectures :)

So, here, for cutemarked, when I want to build it on x86_gcc2 using the x86 secondary arch, I run "haikuporter cutemarked_x86".
It builds fine. I can launch it, but there is no window appearing.

Qt5 is not available on x86_gcc2 primary arch, but it is on x86 secondary arch, so cutemarked should normally run there too... but it does not.
So we should (probably) mark it as untested on x86. Thanks!

Reply:
You mean mark it in secondary arch too?

Reply:
Yes